packages/kit/src/components/TradingView/TradingViewPerpsV2/constants/messageTypes.ts

```
@@ -0,0 +1,6 @@
+export const MESSAGE_TYPES = {
+  MARKS_UPDATE: 'MARKS_UPDATE',
+  MARKS_RESPONSE: 'MARKS_RESPONSE',
+} as const;
+
+export type MessageType = typeof MESSAGE_TYPES[keyof typeof MESSAGE_TYPES];
```

ESLint

[error] Replace `typeof·MESSAGE_TYPES[keyof·typeof·MESSAGE_TYPES];` with `(typeof·MESSAGE_TYPES)[keyof·typeof·MESSAGE_TYPES];⏎`

(prettier/prettier)